specification/storagecache/resource-manager/Microsoft.StorageCache/preview/2020-08-13-aml-preview/storagecache.json

```diff
@@ -2606,10 +2563,8 @@
               "description" : "The state of the archive operation",
               "type" : "string",
               "enum" : [
-                "NotStarted",
-                "InProgress",
-                "Failed",
-                "Complete"
+                "Idle",
```

Comment:
How about adding NotConfigured to indicate that the user has not enabled archive?

Reply:
Its currently NotConfigured, Idle, InProgress

Reply:
In the next commit.

Reply:
I think the enum contents here should be the same as the state in amlFilesystemNamespaceHydrationStatus. Perhaps that should be defined in one place and $ref'd?